### DISCUSSION TITLE
Updating obsolete formula with an underdeveloped successor

### DISCUSSION THREAD

**Discussion by indianets** [Homebrew/discussions#4918] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
.
```

### Output of `brew doctor`

```shell
.
```

### Description of issue

There is a gdrive formula but the author has archived the repository in April, 2023 and has started a new successor which doesn't currently have feature parity.

What is the best way to update in this scenario?

Existing formula `gdrive` in homebrew-core : https://github.com/prasmussen/gdrive
Successor project : https://github.com/glotlabs/gdrive

**Comment by SMillerDev** [comment, marked as answer]
Make a new formula for the new repo
